LOCAL_VIRTUALENV.rst

```diff
@@ -106,7 +106,7 @@ To create and initialize the local virtualenv:
 
 2. Install Python PIP requirements:
 
-   ``pip install -e ".[devel]"``
+   ``pip install -U -e ".[devel,gcp,postgres,ssh]"``
```

Comment:
```suggestion
   ``pip install -U -e ".[devel,<OTHER EXTRAS>]"`` for example: ``pip install -U -e ".[devel,gcp,postgres,ssh]"``   
```

Reply:
The reason I added `postgres` and `gcp`  specifically is because when we first do `airflow db init` it will try to import the example dags (load_examples=yes). Though it can still init the database, the error message can be confusing to first-time contributors. Maybe I should add a note here instead?

Reply:
@xinbinhuang -> yes. I think it makes sense. I think also we should solve it so that the error message is not shown. I even created a JIRA issue for it https://issues.apache.org/jira/browse/AIRFLOW-6265 

I assumed  you close it because you open another request :)?

Reply:
Yeah, I made a mistake by force pushing an empty commit to my branch, and it causes this PR to be closed automatically. I could not find a way to reopen this so I create a new one. Hope it did not cause too many problems